### PR TITLE
Travel journal

### PR DESCRIPTION
**Pull request by Kellypride**
### **New changes made  on branch travel journal.**
1.  Created two components Header and Main. and passed an argument called props.
2.  Props is access in the main component through an object of each properties
3. I created a file data.jsx and stored the properties in an anray.
4. Imported data.js and stored it in a variable called data.  and irritate through each array using data.map()
5. I rendered each comopent in the app.js files

### CODE REVIEW

**Review by Kimbohlovette — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,22 @@
 import Header from './components/Header/Header';
 import Main from './components/Main/Main';
-
+import data from "./data"
 function App() {
```
Comment (sambacarlson) [inline src/App.js:4]: Always good to add an empty line after imports before the rest of the code

File: src/data.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const Items = [
```
Comment (sambacarlson) [inline src/data.jsx:1]: I would recommend that you use `camelCase` for common variables,  and only use `PascalCase` for components. for very important constants, use `CAPITALS`
while `Items` here is a constant, it represents data that will be gotten from a database in future for example. so its better practice to use camel case there

File: src/index.css [hunk 3]
```diff
@@ -1,3 +1,5 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap');
```
Comment (sambacarlson) [inline src/index.css:4]: Add an empty line after the default taildwind directives

**Review by sambacarlson — COMMENTED** [review]
Looks good. I have left some comments

**Pull request merged**